lib/core/types.hpp

```diff
@@ -62,6 +62,18 @@ struct RawAccelerationData {
   const TimePoint measured_at;
 };
 
+struct GyroscopeData {
```

Comment:
What's the use for this? Is there no more suitable alternatives?

Reply:
Shouldn't this be a static thing that can get passed somewhere for full processing?